torchaudio/functional.py

```diff
@@ -9,26 +9,23 @@
     'LC2CL',
     'istft',
     'spectrogram',
-    'create_fb_matrix',
     'spectrogram_to_DB',
+    'create_fb_matrix',
     'create_dct',
-    'BLC2CBL',
     'mu_law_encoding',
     'mu_law_expanding'
 ]
 
 
 @torch.jit.script
 def scale(tensor, factor):
```

Comment:
Do we even still want to keep this one? I mean, it's just "tensor / factor" with an "to(torch.float32)" before.

Reply:
Done.